manifests/charts/ztunnel/values.yaml

```diff
@@ -93,3 +94,16 @@ defaults:
   # TODO Ambient inpod - for OpenShift, set to the following to get writable sockets in hostmounts to work, eventually consider CSI driver instead
   #seLinuxOptions:
   #  type: spc_t
+
+  global:
```

Comment:
I don't mind the concept of supporting globals for use cases where users are composing, but it shoul be done in a way transparent to users not composing (I. e. this values.yaml file should remain unchanged)